lib/uart.toit

```diff
@@ -217,6 +217,57 @@ class Port implements reader.Reader:
       if flushed: return
       sleep --ms=1
 
+
+  static CONTROL_FLAG_LE  ::= 1 << 0            /* line enable */
```

Comment:
It's not a great user experience for embedded programmers to have all these methods and constants that are suggested by the editor's autocomplete, but that don't work because the ESP32 doesn't have support.

How about making a subclass, perhaps called ConfigurablePort, that has the extra constants and methods?